src/difference/difference.js

```diff
@@ -0,0 +1,16 @@
+export const difference = (firstArray, secondArray) => {
```

Comment:
firstArray and secondArray naming are different in the README, they possibly could me more clear?